portal/src/main/webapp/WEB-INF/jsp/global/js_include.jsp

```diff
@@ -104,6 +104,11 @@
 <script type="text/javascript" src="js/lib/jstree.js?<%=GlobalProperties.getAppVersion()%>"></script>
 <script type="text/javascript" src="js/api/cbioportal-client.js?<%=GlobalProperties.getAppVersion()%>"></script>
 
+<%--copy these 3 files from cross_cancer_results.jsp for making expression tab works--%>
```

Comment:
this comment will be confusing when cross_cancer_results.jsp doesn't exist anymore.. maybe just say its for the expression tab